src/main/java/nextstep/core/subway/favorite/application/FavoriteService.java

```diff
@@ -33,20 +32,20 @@ public FavoriteService(FavoriteRepository favoriteRepository, PathFinderService
     }
 
     @Transactional
-    public Favorite createFavorite(FavoriteRequest request, Member member) {
+    public Favorite createFavorite(FavoriteRequest request, Long memberId) {
         if (!pathFinderService.isValidPath(convertToPathFinderRequest(request))) {
             throw new IllegalArgumentException("출발역과 도착역을 잇는 경로가 없습니다.");
         }
-        return favoriteRepository.save(createFavoriteEntity(request, member));
+        return favoriteRepository.save(createFavoriteEntity(request, memberId));
     }
 
     public List<FavoriteResponse> findFavorites(Member member) {
-        return convertToFavoriteResponses(favoriteRepository.findByMember(member));
+        return FavoriteConverter.convertToFavoriteResponses(favoriteRepository.findByMember(member.getId()));
```

Comment:
이와 같이 코드가 길어지는 경우 가독성을 해칠 수 있는 것 같아요.
개인적으로는 아래와 같이 풀어쓰는 방식이 가독성이 좋다고 생각해요!

```suggestion
        List<Favorite> favorites = favoriteRepository.findByMember(member.getId());
        return FavoriteConverter.convertToFavoriteResponses(favorites);
```

Reply:
앞으로는 이렇게 코드를 작성하지 않을 것 같습니다. 😢 
(극단적인 라인 줄이기...는 하지 않겠다.. 메모..!!)